commands/course.js

```diff
@@ -123,18 +123,6 @@ module.exports = {
                         ? `${course} (alias for \`${input_course}\`)`
                         : `${course}`;
 
-                // Check if the member already has an entry in the channel's permission overwrites, and update
```

Comment:
Uh i fixed this in a different way and pushed it. So perhaps pull from develop again